Languages/IronPython/Tests/modules/system_related/thread_test.py

```diff
@@ -195,7 +195,7 @@ def tempFunc():
     thread.start_new(tempFunc, ())
     while CALLED==False:
         print ".",
-        time.sleep(1)
```

Comment:
If you could retain the call to time.sleep, just change to 0.1 as the parameter.